ee/app/license/server/license.ts

```diff
@@ -228,6 +229,14 @@ export function onLicense(feature: string, cb: (...args: any[]) => void): void {
 	EnterpriseLicenses.once(`valid:${ feature }`, cb);
 }
 
+export function onValidateLicenses(cb: (...args: any[]) => void): void {
+	if (isEnterprise()) {
+		return cb();
+	}
+
+	EnterpriseLicenses.once('validate', cb);
```

Comment:
What happens if the admin changes the license in run time? Wouldn't be better to use `on`?

Reply:
The callback is only added when there was no license already loaded, but I changed it to `on` anyway because otherwise it would not work if the user typed an invalid license and then a valid one.